docs/topics/signals.txt

```diff
@@ -190,6 +190,14 @@ In some circumstances, the code connecting receivers to signals may run
 multiple times. This can cause your receiver function to be registered more
 than once, and thus called multiple times for a single signal event.
 
+Although `:meth:`~django.apps.AppConfig.ready` is designed to be called
+only once during Django's startup sequence, unit test runs may break this
+implicit contract. Another (albeit less likely) cause for this is if
+you import your handlers.py module from somewhere in your project; the signal
```

Comment:
`handlers.py` is not _necessarily_ where such things exist. A vague convention might exist (but could also just as easily be `listeners.py` or `signals.py` if someone so wished) but it's not a requirement, and whether or not those imports are OK from the point of view of signal registration depends entirely if they perform their _registration_ there, or just _exist_ there (precisely because registration can and probably should be deferred until the AppConfig's `ready()` method is called).

Or, to summarise. Don't say `handlers.py` (as far as I can tell, it's not mentioned elsewhere in this document)

Reply:
Looks like there is wording related to `signals` submodule (as a convention), but good call -- it isn't worth duplicating/binding/confusing users, when other parts of the docs already make suggestions (not assumptions). I've made the update.